src/App.scss

```
@@ -0,0 +1,10 @@
+$bqcream: #FCFAE3;
+$bqpink: #FF7BAC;
+$bqgreen: #1EBC9E;
+$btMenu: '../src/Assets/btnMenu.png';
+
+
+.content{
```

¿en que componente se va a usar este content????
```suggestion
.content-component{
```